Frontend/android-native/mobility-driver/src/main/java/in/juspay/mobility/driver/MobilityDriverBridge.java

```diff
@@ -456,18 +456,20 @@ public void renderBase64ImageFile(String base64Image, String id, boolean fitCent
             try {
                 if (!base64Image.equals("") && id != null && bridgeComponents.getActivity() != null) {
                     LinearLayout layout = bridgeComponents.getActivity().findViewById(Integer.parseInt(id));
-                    byte[] decodedString = Base64.decode(base64Image, Base64.DEFAULT);
-                    Bitmap decodedByte = BitmapFactory.decodeByteArray(decodedString, 0, decodedString.length);
-                    ImageView imageView = new ImageView(bridgeComponents.getContext());
-                    imageView.setImageBitmap(decodedByte);
-                    if (fitCenter) {
-                        imageView.setScaleType(ImageView.ScaleType.FIT_CENTER);
-                    } else {
-                        imageView.setScaleType(ImageView.ScaleType.CENTER_CROP);
-                    }
-                    imageView.setAdjustViewBounds(true);
-                    imageView.setClipToOutline(true);
-                    if (layout != null) {
+                    if (layout != null){
+                        byte[] decodedString = Base64.decode(base64Image, Base64.DEFAULT);
+                        Bitmap decodedByte = BitmapFactory.decodeByteArray(decodedString, 0, decodedString.length);
+                        ImageView imageView = new ImageView(bridgeComponents.getContext());
+                        ViewGroup.LayoutParams layoutParams = new ViewGroup.LayoutParams(layout.getWidth(),layout.getHeight());
+                        imageView.setLayoutParams(layoutParams);
+                        imageView.setImageBitmap(decodedByte);
+                        if (fitCenter) {
+                            imageView.setScaleType(ImageView.ScaleType.FIT_CENTER);
+                        } else {
+                            imageView.setScaleType(ImageView.ScaleType.CENTER_CROP);
+                        }
```

Comment:
to make it more configurable we can send `scaleType` as String and use `getScaleTypes`